Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -543,6 +546,9 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     RefPtr<WebCore::GraphicsLayer> m_scrolledContentsLayer;
     RefPtr<WebCore::GraphicsLayer> m_pageBackgroundsContainerLayer;
     RefPtr<WebCore::GraphicsLayer> m_contentsLayer;
+#if ENABLE(UNIFIED_PDF_PAINT_SELECTIONS)
+    RefPtr<WebCore::GraphicsLayer> m_selectionsLayer;
```

Comment:
`m_selectionLayer`

Reply:
Addressed in https://github.com/WebKit/WebKit/commit/3175c379f8966936bbf4d9833ad4b1c11e7a1ede